tests/integration/targets/activation/tasks/main.yml

```diff
@@ -360,44 +384,106 @@
           - _result.failed
           - "'Organization Invalid_Organization not found.' in _result.msg"
 
-    - name: Create a new rulebook activation again
+    - name: Create a new rulebook activation again (no updates)
       ansible.eda.rulebook_activation:
         name: "{{ activation_name }}"
         description: "Example Activation description"
         project_name: "{{ project_name }}"
         rulebook_name: "{{ rulebook_name }}"
         decision_environment_name: "{{ decision_env_name }}"
         enabled: False
+        eda_credentials:
+          - "{{ credential_name_aap }}"
         organization_name: Default
       register: _result
 
-    - name: Check rulebook activation creation
-      assert:
-        that:
-          - not _result.changed
-          - "'A rulebook activation with name: ' + activation_name + ' already exists. The module does not support modifying a rulebook activation.' in _result.msg"
-
     - name: Get information about the rulebook activation
       ansible.eda.rulebook_activation_info:
         name: "{{ activation_name }}"
       register: _result_activation_info
 
-    - name: Assert that rulebook activation is disabled
-      ansible.builtin.assert:
+    - name: Check rulebook activation not updated
+      assert:
         that:
-          - not _result_activation_info.activations[0].is_enabled
+          - not _result.changed
 
-    - name: List all the rulebook activations
+    - name: Disable rulebook activation
+      ansible.eda.rulebook_activation:
+        name: "{{ activation_name }}"
+        state: disabled
+
+    - name: Wait until the activation is disabled
+      register: activation_status
+      until:
+        - not (activation_status.activations | selectattr('name', 'equalto', activation_name) | list | first).is_enabled
+        - "'stopped' == (activation_status.activations | selectattr('name', 'equalto', activation_name) | list | first).status"
+      retries: 30
+      delay: 2
       ansible.eda.rulebook_activation_info:
 
-    - name: Delete rulebook activation
+    - name: Disable rulebook activation again
       ansible.eda.rulebook_activation:
         name: "{{ activation_name }}"
-        state: absent
+        state: disabled
+
+    - name: Check rulebook activation not updated
+      assert:
+        that:
+          - not _result.changed
+
+    - name: Update rulebook activation name and fields
+      ansible.eda.rulebook_activation:
```

Comment:
@kaiokmo Can we add or update a test where we modify the event streams field?

Reply:
done